src/database/config/config.js

```diff
@@ -0,0 +1,17 @@
+
+import dotenv from 'dotenv';
+dotenv.config();
+
+export const development = {
+  url: process.env.DEV_DATABASE_URL,
+  dialect: 'postgres',
+}
+export const test = {
+  url: process.env.TEST_DATABASE_URL,
+  dialect: 'postgres',
+}
+export const production = {
+  url: process.env.DATABASE_URL,
+  dialect: 'postgres',
+}
+
```

Comment:
Why three configurations file? I see you have this configuration in `database/config/config.js`, `src/config/config.js` and another one in `src/config.js`

Reply:
I have to remove the configuration in database folder and src/config.js , the useful one is src/config/config.js